src/tracing/CMakeLists.txt

```diff
@@ -43,9 +43,9 @@ endfunction()
 
 set(osd_traces oprequest.tp osd.tp pg.tp)
 add_tracing_library(osd_tp "${osd_traces}" 1.0.0)
-add_tracing_library(rados_tp librados.tp 2.0.0)
+add_tracing_library(librados_tp librados.tp 2.0.0)
```

Comment:
@renzhengeek if only we are able to root cause
```
    /var/ceph/ceph/src/librados/librados.cc:47:30: fatal error:
    tracing/librados.h: No such file or directory
     #include "tracing/librados.h"
                                  ^
    compilation terminated.
```
and understand why reverting 0a71759 helps, i am happy to do so.